src/status_im2/contexts/chat/messages/content/view.cljs

```diff
@@ -141,18 +141,19 @@
               [status/status outgoing-status])]]]]))))
 
 (defn message-with-reactions
-  [{:keys [pinned-by mentioned in-pinned-view? content-type last-in-group? message-id] :as message-data}
-   {:keys [chat-id] :as context}
-   keyboard-shown]
-  [rn/view
-   {:style               (style/message-container in-pinned-view? pinned-by mentioned last-in-group?)
-    :accessibility-label :chat-item}
-   (when pinned-by
-     [pin/pinned-by-view pinned-by])
-   (if (#{constants/content-type-system-text constants/content-type-community
-          constants/content-type-system-pinned-message
-          constants/content-type-contact-request}
-        content-type)
-     [system-message-content message-data]
-     [user-message-content message-data context keyboard-shown])
-   [reactions/message-reactions-row chat-id message-id]])
+  [{:keys [pinned-by mentioned in-pinned-view? content-type last-in-group?] :as message-data}
+   context]
+  (let [user-message-content-instance (reagent/as-element [user-message-content message-data context])
```

Comment:
could you elaborate why we need this in let and as element?

Reply:
This improved the performance of the message rendering when a user long presses a reaction, It renderes once and is saved to the `user-message-content-instance` var and passed to the child component, Instead of making the component redo the rendering process.

Reply:
sounds strange

Reply:
The message is already shown, We just need to show it as the `selected-item` in `bottom-sheet`

Reply:
Reverted